driverkit/utils/generate

```diff
@@ -52,6 +62,49 @@ compute_kernelrelease() {
     TARGET_KERNEL_RELEASE="${TARGET_KERNEL%_${TARGET_KERNEL_VERSION}}"
 }
 
+check_outputs() {
+    regex='([0-9]+)\.([0-9]+)'
+    if [[ "${TARGET_KERNEL}" =~ ${regex} ]]
+    then
+        maj=${BASH_REMATCH[1]}
+        min=${BASH_REMATCH[2]}
+
+        # Use correct array for target arch
+        declare -n arrayname=${TARGET_ARCH}_MINIMUM_KVER
+        ctr=0
+        for min_kver in "${arrayname[@]}" ; do
+            KEY="${min_kver%%:*}"
+            VALUE="${min_kver##*:}"
+
+            if [[ "${VALUE}" =~ ${regex} ]]
+            then
+                req_maj=${BASH_REMATCH[1]}
+                req_min=${BASH_REMATCH[2]}
```

Comment:
Take required major and minor versions, ie: the minimum kernel version where we support eBPF and kmod.